2018/Chuvagina/NewTournament/TournamentUI/MainWindow.xaml.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
```

Comment:
It seems that UI is not finished. If you have questions feel free in asking. Also if i'm not mistaken you used WPF technology for main window and WinForms for small dialog. Please choose one technology for all UI controls.